app/js/views/new_todo_view.js

```diff
@@ -0,0 +1,31 @@
+define([
+  'jquery',
+  'backbone',
+  'text!templates/new_todo_view.ejs',
+  'model/todo',
+  'collection/todos'
+], function ($, Backbone, indexTemplate, ToDo, ToDos) {
```

Comment:
Obviosly this is not indexTemplate anymore, [copy paste cats](http://memegenerator.net/instance/38281367)